meta/main.yml

```diff
@@ -37,5 +36,3 @@ galaxy_info:
         - focal
         - jammy
   role_name: python
-
-dependencies: []
```

Comment:
This `dependencies` block was supposed to move to the top of the file, not be deleted.

Reply:
Resolved in commit b4ed7b0.